src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int QUANTITY_INDEX = 1;
+    private static final int ACTION_INDEX = 0;
```

Comment:
dont invent new names
![image](https://github.com/mate-academy/jv-work-with-file/assets/73041091/76493f38-1f71-4433-9440-6b8b71699ec9)
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
    private static final int AMOUNT_INDEX = 1;
```